### PR TITLE
Add test_bindings template

### PR DESCRIPTION
**Pull request by mdellweg**
[noissue]

### CODE REVIEW

**Comment by mdellweg** [comment]
There is improvement to be done:
https://travis-ci.com/pulp/pulp_deb/jobs/212349703#L469

**Comment by bmbouter** [comment]
https://github.com/pulp/plugin_template/pull/26

File: templates/travis/.travis/test_bindings.py.j2 [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# TODO: Plugin writer action required
+# This templated file must be adjusted to the plugins special dietary needs.
+
+from pulpcore.client.pulpcore import (
+    ApiClient as CoreApiClient,
+    ArtifactsApi,
+    Configuration,
+    Repository,
+    RepositoriesApi,
+    RepositoriesVersionsApi,
```
Comment (fabricio-aguiar) [inline templates/travis/.travis/test_bindings.py.j2]: Repositories are implemented on plugins now

**Comment by daviddavis** [comment]
Note that [we had to revert this change](https://github.com/pulp/plugin_template/pull/173). The Travis template files are synchronized (not just bootstrapped) and this file was overwriting the test_bindings.py in pulpcore and other plugins.

Longer term, the goal is to get rid of this file and [use bindings in our tests](https://pulp.plan.io/issues/6020).

**Pull request merged**